### PR TITLE
Replace the flash player with medialement.js for the shared media

### PR DESCRIPTION
**Pull request by jvoisin**

### CODE REVIEW

**Comment by jvoisin** [comment]
This is related to #868

**Comment by jvoisin** [comment]
It's currently looking like this:

x

File: airsonic-main/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp [hunk 1]
```diff
@@ -4,68 +4,17 @@
 <html>
 <head>
     <%@ include file="head.jsp" %>
-    <script type="text/javascript" src="<c:url value="/script/swfobject.js"/>"></script>
     <script type="text/javascript" src="<c:url value="/script/prototype.js"/>"></script>
-
-    <meta name="og:type" content="album"/>
```
Comment (heyarne) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp]: Any reason for removing this particular meta tag?

Comment (jvoisin) [reply airsonic-main/src/main/webapp/WEB-INF/jsp/externalPlayer.jsp]: Nope, I'll add it back.

**Comment by jooola** [comment]
We forgot to clean the flash libraries.

**Comment by jvoisin** [comment]
Be my guest ♥

**Comment by almet** [comment]
Thanks @jvoisin :-)

**Pull request merged**